src/lib/uTop_private.ml

```diff
@@ -9,6 +9,28 @@
 
 open Lwt_react
 
+module DefaultPaths = struct
+  let ( / ) = Filename.concat
+  let xdg = Xdg.create ~env:Sys.getenv_opt ()
+
+  (* Fallback to legacy filename *)
+  let resolve legacy fn =
+    if not (Sys.file_exists fn) && Sys.file_exists legacy then
+      legacy
+    else
+      fn
+
+  let history_file_name =
```

Comment:
can this, and `config_file_name`, be functions instead, to avoid IO at module load time?